spec/services/shopify_import/products/create_spec.rb

```diff
@@ -57,11 +57,28 @@
         end
       end
 
+      context 'spree variants' do
+        let(:spree_product) { Spree::Product.find_by!(name: shopify_product.title) }
```

Comment:
nitpick: you could find these by a handle which is indexed, rather than by a title